inst/rmarkdown/templates/template_sfp_en/skeleton/13_subprotocols.Rmd

```diff
@@ -0,0 +1,20 @@
+# (PART) Subprotocols {-} 
+
+```{r, results="asis"}
+if (nrow(dependencies) > 0) {
+  mdfiles <- paste0(dependencies$protocol_code,
+                    "-", 
+                    dependencies$version_number, 
+                    ".md")
+  child_docs <- file.path(dependencies$version_number,
+                          mdfiles)
+  
+  child_docs <- child_docs[dependencies$appendix]
+  
+  if (length(child_docs) > 0) {
+    res <- map(child_docs, knit_child, quiet = TRUE)
+    cat(unlist(res), sep = "\n")
+  }
```

Comment:
Maybe you could add an `else` to make appear something like 'No subprotocols included' below the Subprotocols part title (since the latter will always be there). On the other hand, I'm not sure whether you can actually see such text (without chapter title above it) in the case of a gitbook. Feel free to ignore.

Reply:
See [special headers](https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html#special-headers): 
> A part title should be written right before the first chapter title in this part, both title in the same document.

So I'm guessing this will not be possible.